main.css

```diff
@@ -1,61 +1,77 @@
-html,
-body {
-    font-family: sans-serif;
-    margin: 0px;
-}
-
 /* header */
-
 .header {
     display: flex;
-    height: 220px;
-    border-bottom: 0.5px solid grey;
-    justify-content: center;
+    height: 80px;
+    border-bottom: 0.5px solid #D8D8DE;
+    justify-content: space-between;
     align-items: center;
-    padding-left: 80px;
-    padding-right: 80px;
+    padding: 0px 15px 0px 6px;
 }
 
 .header__glass {
     position: relative;
-    bottom: 100px;
-    left: 40px;
+    bottom: 35px;
```

Comment:
why relative? can we use margin instead?

Reply:
+